internal/cmd/asyncapi/command_export_test.go

```diff
@@ -313,38 +15,6 @@ func TestHandlePrimitiveSchemas(t *testing.T) {
 	require.Error(t, err)
 }
 
-func TestGetBindings(t *testing.T) {
-	c, err := newCmd()
```

Comment:
Thanks for rewriting these as integration tests! They'll be much easier to maintain in the future.